src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTaskTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.tasks;
+
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.IntegrationTest;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+import static org.awaitility.Awaitility.await;
+import static org.mockito.Mockito.verify;
+
+@RunWith(SpringRunner.class)
```

Comment:
We should try to convert everything to Junit5. Most of the tests are already using it